tests/manage/z_cluster/nodes/test_node_replacement_proactive.py

```diff
@@ -97,4 +99,8 @@ def test_nodereplacement_proactive(self, pvc_factory, pod_factory, dc_pod_factor
         self.sanity_helpers.delete_resources()
         # Verify everything running fine
         log.info("Verifying All resources are Running and matches expected result")
-        self.sanity_helpers.health_check()
+        self.sanity_helpers.health_check(cluster_check=False)
+        log.info("Verifying ceph cluster status")
+        ceph_health_check(
+            namespace=config.ENV_DATA['cluster_namespace'], tries=10,
```

Comment:
You dont need to specify namespace here as by default the function takes ocs namespace